livy/src/main/java/org/apache/zeppelin/livy/BaseLivyInterpreter.java

```diff
@@ -95,6 +96,13 @@ public BaseLivyInterpreter(Properties property) {
     this.pullStatusInterval = Integer.parseInt(
         property.getProperty("zeppelin.livy.pull_status.interval.millis", 1000 + ""));
     this.restTemplate = createRestTemplate();
+    if (!StringUtils.isBlank(property.getProperty("zeppelin.livy.http.headers"))) {
+      String[] headers = property.getProperty("zeppelin.livy.http.headers").split(";");
+      for (String header : headers) {
+        String[] splits = header.split(":");
+        customHeaders.put(splits[0].trim(), splits[1].trim());
```

Comment:
handle the case when `splits[1]` is empty?

Reply:
per rfc, looks like `:` is required but field value is not.
we should probably error out if `:` is missing but handle empty value

Reply:
Fixed